apps/E2E/src/Text/pages/TextPageObject.ts

```diff
@@ -1,12 +1,56 @@
+import { Attribute } from '../../common/consts';
 import {
   TEXT_TESTPAGE,
   FIRST_TEXT_COMPONENT,
   HOMEPAGE_TEXT_BUTTON,
   SECOND_TEXT_COMPONENT,
+  THIRD_TEXT_COMPONENT,
+  FOURTH_TEXT_COMPONENT,
 } from '../../../../fluent-tester/src/TestComponents/Text/consts';
 import { BasePage, By } from '../../common/BasePage';
 
+export const enum TextComponentSelector {
+  First = 0, // this._firstComponent
+  Second, // this._secondComponent
+  Third, // this._thirdComponent
+  Fourth, // this._fourthComponent
+}
+
 class TextPageObject extends BasePage {
+  /* Gets the accessibility role of an UI element given the selector */
+  async getTextAccessibilityRole(componentSelector: TextComponentSelector): Promise<string> {
```

Comment:
To enable code-reuse, let's create a function called `getTextComponent(componentSelector: TextComponentSelector) : Promise<WebdriverIO.Element>` and put the switch statement in there.

That way, in `getTextAccessibilityRole()` and `getTextAccessibilityLabel()`, we can just call 
`return await (this.getElementAttribute(await this.getTextComponent(componentSelector), Attribute.AccessibilityRole)`